keps/sig-network/20191205-kube-proxy-IP-node-binding.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Give the ability to the LoadBalancer provisioner or a user to disable the LB IP binding
+authors:
+  - "@Sh4d1"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cloud-provider
+reviewers:
+  - "@thockin"
+  - "@cheftako"
+  - TBD
+approvers:
+  - "@thockin"
+  - TBD
+editor: TBD
+creation-date: 2019-12-05
+last-updated: 2019-12-21
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Give the ability to the LoadBalancer provisioner or a user to disable the LB IP binding
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+<!-- /toc -->
+
+## Summary
+
+The different kube proxy implentations (at least ipvs and iptables) are today sort of binding the External IPs of the LoadBalancer Service to each node. (iptables is creating some rules to redirect directly the packets to the service and ipvs is binding the IP to one interface on the node). This feature is exisiting because:
```

Comment:
typos:

The different ~kube proxy~ `kube-proxy` ~implentations~ `implementations` (at least ipvs and iptables)`, as of today, are` ~are today sort of~ binding the External IPs of the LoadBalancer Service to each node ~.~ (iptables ~is~ `are` creating some rules to redirect ~directly the packets~ `packets directly` to the service and ipvs ~is~ `are` binding the IP to one interface on the node). This feature ~is exisiting~ `exists` because: